mkdocs.yml

```diff
@@ -72,9 +72,10 @@ nav:
       - 'Python with pip': build-install/pip.md
     - 'Debugging and Performance Analysis':
       - software/devtools/index.md
-      - 'Linaro Forge uenv': software/devtools/linaro-forge.md
-      - 'Linaro MAP': software/devtools/linaro-map.md
-      - 'Linaro DDT': software/devtools/linaro-ddt.md
+      - 'Linaro Forge': 
```

Comment:
One of our goals was to flatten the hierarchy of the docs as much as possible, and specifically reduce the number of levels in the docs.

I see why you add this additional layer, but I would personally prefer to keep all of the pages in here at one level (so every tool is on level three of the index), and focus on making sure that the main `devtools` index is well organised.

Reply:
Yes, as I mentioned this was a personal preference (since "Linaro Forge" is a page one should read before "Linaro DTT" and "Linaro MAP), but I didn't really follow the previous discussion and I'm happy to bring the pages back to their original position.